jerry-core/parser/js/js-parser-limits.h

```diff
@@ -32,10 +32,18 @@
 #define PARSER_MAXIMUM_IDENT_LENGTH 255
 #endif /* !PARSER_MAXIMUM_IDENT_LENGTH */
 
+/* Maximum string limit.
+ * Limit: 2147483647 / 65535. */
+#ifdef JERRY_CPOINTER_32_BIT
+#define PARSER_MAXIMUM_STRING_LIMIT 2147483647
+#else /* !JERRY_CPOINTER_32_BIT */
+#define PARSER_MAXIMUM_STRING_LIMIT 65535
+#endif /* JERRY_CPOINTER_32_BIT */
+
 /* Maximum string length.
- * Limit: 65535. */
+ * Limit: PARSER_MAXIMUM_STRING_LIMIT. */
 #ifndef PARSER_MAXIMUM_STRING_LENGTH
-#define PARSER_MAXIMUM_STRING_LENGTH 65535
+#define PARSER_MAXIMUM_STRING_LENGTH PARSER_MAXIMUM_STRING_LIMIT
```

Comment:
If it is the same as the other define, do we need it at all?

Reply:
Yes, because the maximum is the default configuration, although we allow lower values.

Reply:
I see